packages/extension/src/libs/background/index.ts

```
@@ -172,6 +175,9 @@ class BackgroundHandler {
       case InternalMethods.ethereumDecrypt:
         return ethereumDecrypt(this.#keyring, message);
       case InternalMethods.unlock:
+        const password = message?.params?.[0] as string;
```

Biome (2.4.11)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)